nextjs-client/src/components/Layout/style.js

```diff
@@ -1,7 +1,10 @@
 import styled from '@emotion/styled';
 
 export const Container = styled.div`
-  width: 100vw;
   height: 100vh;
-  background-color: gray;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
```

Comment:
Do we need `justify-content` here?

Reply:
We don't need `justify-content`. It was the default styling when I set up the Layout component.